### PR TITLE
Built worker image on top of the centos:8

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
The sentry-sdk and rebase-helper packages are installed from GitHub via pip.

Signed-off-by: Frantisek Lachman <flachman@redhat.com>

### CODE REVIEW

**Comment by jpopelka** [comment]
Can you explain why we install sentry-sdk and rebase-helper from github instead of PyPI?

**Comment by lachmanfrantisek** [comment]
> Can you explain why we install sentry-sdk and rebase-helper from github instead of PyPI?

I've included some explanation, but the main problem is that for some reason, the `rebasehelper` can't be installed from PyPI during the built event it works well when I do the steps manually inside a container.

File: files/install-deps-worker.yml [hunk 1]
```diff
@@ -12,16 +12,28 @@
...
       pip:
         name:
-          - sentry-sdk
+          # Using GitHub because
+          # a) rebasehelper has some problem being installed from PyPI
+          #    (event it works from inside a container)
+          # b) We can freeze the source-code version
+          #    and choose any git ref of the packages.
+          - git+https://github.com/getsentry/sentry-python@0.19.1
+          - git+https://github.com/rebase-helper/rebase-helper@0.23.1
```
Comment (jpopelka) [inline files/install-deps-worker.yml:36]: I don't mind this solution, just FYI that
```yaml
          - sentry-sdk==0.19.1
          - rebasehelper==0.23.1
```
works OK for me

**Review by jpopelka — APPROVED** [review]

File: files/install-deps-worker.yml (hunk 1 above)
Comment (lachmanfrantisek) [reply files/install-deps-worker.yml:36]: I know the syntax, but wanted to use `rebasehelper` first and it did not build. I'll try with the version and see if it helps.

Comment (lachmanfrantisek) [reply files/install-deps-worker.yml:36]: Ok, it works.

**Comment by csomh** [comment]
@lachmanfrantisek would it make sense to base the worker image on top of the dist2src image built from [Containerfile](https://github.com/packit/dist-git-to-source-git/blob/d0510bd22879419871ab489e1ba51b18188c30e3/Containerfile)?

When a new dependency is needed to make conversion work, we update that one. Workers need the same dependencies.

**Comment by lachmanfrantisek** [comment]
@csomh Yes, it makes sense. Will send a PR.

We might want to move some logic to the playbook as well, but that's a different task.

**Comment by lachmanfrantisek** [comment]
@csomh It looks like we are not building that image in docker-hub -- I'll create an issue instead.

**Pull request merged**